refinery/tool_manager/models.py

```diff
@@ -400,6 +416,33 @@ def update_file_relationships_with_urls(self):
             )
         self.set_tool_launch_config(tool_launch_config)
 
+    def is_running(self):
+        if self.is_workflow():
+            return True if self.analysis.running() else False
+
+        try:
+            self._django_docker_client.lookup_container_url(
+                self.container_name
+            )
+            return True
+        except NotFound:
+            return False
```

Comment:
Sorry: There should be a better method for this. I've filed https://github.com/refinery-platform/django_docker_engine/issues/94, but obviously not a blocker for this PR.